dotCMS/src/main/java/com/dotmarketing/business/VersionableFactoryImpl.java

```diff
@@ -194,27 +197,37 @@ protected Versionable findDeletedVersion(String id) throws DotDataException, Dot
 			throw new DotDataException("identifier:" + identifier +" not found");
 		}
 
-		final Class<?> clazz = InodeUtils.getClassByDBType(identifier.getAssetType());
-		if(clazz.equals(Inode.class)) {
+		if (identifier.getAssetType().equals("contentlet")){
```

Comment:
Always the constant to the left 

"contentlet".equals(identifier.getAssetType())

Reply:
Done